src/ripple/test/jtx/Env.h

```diff
@@ -60,38 +69,37 @@ namespace detail {
 // list or non-static data member initializer is
 // not implemented
 //
-template <std::size_t N>
-struct noripple_helper
+template <class T, std::size_t N>
+struct vararg_array_helper
 {
-    std::array<Account, N> args;
+    std::array<T, N> args;
 
-    template <class T, class ...Args>
-    std::array<Account, sizeof...(Args)>
+    template <class U, class ...Args>
+    std::array<T, sizeof...(Args)>
     flatten (Args&& ...args)
     {
-        return std::array<T,
-            sizeof...(Args)> {
-                std::forward<Args>(args)...};
+        return std::array<U,sizeof...(Args)> {
+            std::forward<Args>(args)...};
```

Comment:
@vinniefalco @nbougalis Confirmed that the `std::array` change proposed above works on windows.